src/app/tabs/settings/email-popover/email-popover.page.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, Validators } from "@angular/forms";
+import { PopoverController } from '@ionic/angular';
+import { first, map, tap, switchMap } from 'rxjs/operators';
+import { DataStoreService } from 'src/app/core/services/data-store.service';
+
+@Component({
+  selector: 'app-email-popover',
+  templateUrl: './email-popover.page.html',
+  styleUrls: ['./email-popover.page.scss'],
+})
+export class EmailPopoverPage implements OnInit {
+
+  emailForm = this.formBuilder.group({
+    email: ['', Validators.email]
+  });
+  email$ = this.dataStoreService.userData$
```

Comment:
The observable is used to perform a side effect (update value of the email form).
We can use <ng-container> and async pipe to manage its subscription even if we don't need to display anything from it.